src/Microsoft.VisualStudio.ProjectSystem.Managed/Composition/OrderPrecedenceImportCollectionExtensions.cs

```diff
@@ -56,7 +58,7 @@ internal static IEnumerable<T> ExtensionValues<T>(this OrderPrecedenceImportColl
             }
         }
 
-        public static T FirstOrDefaultValue<T>(this OrderPrecedenceImportCollection<T> imports, Func<T, bool> predicate)
+        public static T? FirstOrDefaultValue<T>(this OrderPrecedenceImportCollection<T> imports, Func<T, bool> predicate) where T : class
```

Comment:
Had to add a `class` type constraint here in order to return `default`. This is fine for our current usages but feels a little unnecessarily restrictive.